src/constants.h

```diff
@@ -104,6 +104,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 // the file attempting to ensure a unique filename
 #define SCREENSHOT_MAX_SERIAL_TRIES 1000
 
+// Maximum speed (absolute value) for active objects.
+#define OBJECT_MAX_SPEED 500
```

Comment:
please set it upper, if someone want to add laser or projectiles this could be too low